velox/connectors/hive/storage_adapters/s3fs/S3FileSystem.cpp

```diff
@@ -137,15 +138,24 @@ namespace filesystems {
 
 namespace S3Config {
 namespace {
+// S3 configuration parameters are used as keys into the config map.
 constexpr char const* kPathAccessStyle{"hive.s3.path-style-access"};
 constexpr char const* kEndpoint{"hive.s3.endpoint"};
 constexpr char const* kSecretKey{"hive.s3.aws-secret-key"};
 constexpr char const* kAccessKey{"hive.s3.aws-access-key"};
 constexpr char const* kSSLEnabled{"hive.s3.ssl.enabled"};
+constexpr char const* kS3IAMRole{"hive.s3.iam-role"};
+constexpr char const* kS3IAMRoleSessionName{"hive.s3.iam-role-session-name"};
 constexpr char const* kUseInstanceCredentials{
     "hive.s3.use-instance-credentials"};
 } // namespace
 } // namespace S3Config
+namespace S3ConfigDefaults {
+namespace {
+// Default values for S3 configuration parameters.
+const std::string kS3IAMRoleSessionName{"presto-session"};
```

Comment:
Velox aims to be engine agnostic. What's the meaning of this value? Can it changed to velox-session?

Reply:
Yes, can be `velox-session`. The RoleSessionName is used as an identifier. Some uses of it are described below.
https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html
Will fix.